docs/develop/sdk/1-getting-started-guide/00-getting-started.md

```diff
@@ -0,0 +1,179 @@
+The guide requires some experience with javascript and command-line tools.
+We will have recommended guides to other tutorials for further learnings.
+
+## Setup
+
+We will focus on creating a new project from stratch, which will require a little setup.
+First, we need to create a new project in a new directory. For this, we run `mkdir kilt-rocks && cd kilt-rocks`.
+
+From inside the `kilt-rocks` project directory, install the **SDK**, **node** and **node-fetch** with either of the following package managers:
+
+```bash npm2yarn
+npm install @kiltprotocol/sdk-js node node-fetch
+```
+
+After you have imported the SDK, you are now able to access the functionality of KILT.
+With all the required dependencies set, just create a new (empty) script file with `touch getting-started.js`.
+Inside the `package.json` add in the object `"type": "module"`.
+
+Lets first create an asynchronous function called `main` in order to excute the script.
+Underneath the first line add the following:
+
+<CodeBlock className="language-js">
+  {Example1}
+</CodeBlock>
+
+If the setup is correct you can excute the script by calling the name of the file using node.
+
+```bash npm2yarn
+npm node getting-started.js
+```
+ 
+
+You can always excute this file with the command.
+It will refer to this command when requested to excute.
+
+**Let's get started 🔥.**
+
+### Importing KILT's SDK into a project
+
+Lets begin by importing the **SDK** and **node-fetch** into the `getting-started.js`.
+
+``` js
+import * as Kilt from '@kiltprotocol/sdk-js'
+import fetch from 'node-fetch'
+```
+
+Now you are able to access the SDK and all its functionality.
+We will move onto connecting to the **KILT blockchain**.
+
+### Connecting to KILT blockchain
+
+Connecting and disconnecting to the KILT is an important step requiring a setup of the configurations.
+The connection allows to **query**, **verify** and **interact** with the chain.
+
+**Lets get connected.**
+
+Inside the `getting-started.js` inside the `main` function, you will need to begin by connecting to a **KILT node**.
+Using the imported SDK, it exposes **`Kilt.init()`** to initalise the connection to the KILT blockchain via an address.
+
+We will initalise the **KILT blockchain** named the **Spiritnet**.  
+
+<CodeBlock className="language-js">
+  {Example2}
+</CodeBlock>
+
+Now you have initalised the connection, lets connect to the chain.
+Inside the `main` function, lets get the conncetion using an asynchronous call that creates a connection, or checks if an existing connection is available and connects directly.
+
+<CodeBlock className="language-js">
+  {Example3}
+</CodeBlock>
+
+Now you have connected you have access to the chain, but lets not forget to **close** any connections.
+Its best practice not to leave an connections open, add `Kilt.disconnect()` at the bottom of `main` function.
+
+<CodeBlock className="language-js">
+  {Example10}
+</CodeBlock>
```

Comment:
I would suggest to have a single code block with all three methods and just go over it line by line. That way it's easy to copy&past and you have a the full workflow together in an example.

Reply:
I prefer it in this manner, since you are breaking every step down and you don't have huge block of text explaining this.